docs/elements/hx-search-assistance/index.html

```diff
@@ -1,9 +1,10 @@
 ---
 title: <hx-search-assistance>
-minver: 0.6.0
+minver: 0.18.0
```

Comment:
Please revert `minver`. The `<hx-search-assistance>` element hasn't changed.

Reply:
Reverted `<hx-search-assistance>` `minver` metadata to `v0.6.0`.